src/writeData/utils/parsePluginsConfigurationText.mjs

```diff
@@ -488,12 +529,12 @@ getVersion.then(version => {
               )
             } else {
               console.warn(
-                logSymbols.success +
-                  ` All Windows Plugins are listed${
-                    windowsPluginsNotUpdated.length
-                      ? ' but may not be updated '
-                      : ' '
-                  }in the Data > Sources page of the UI\n`
+                logSymbols.success + ' \x1b[32m%s\x1b[0m',
+                ` All plugins are accounted for${
+                  windowsPluginsNotUpdated.length
+                    ? ' but may not be updated '
+                    : ' '
+                }in the Load Data > Sources page of the UI\n`
```

Comment:
this looks like a success message. should it be a `console.warn` or a `console.log`?

Reply:
I was trying to avoid linter complaints, but it looks like that is not necessary.

Updated to `console.log`